package.json

```diff
@@ -48,6 +49,7 @@
   "resolutions": {
     "@definitelytyped/typescript-versions": "^0.0.40",
     "@typescript-eslint/eslint-plugin": "^4.12.0",
+    "@effection/core": "^2.2.0",
```

Comment:
Unfortunately we get a version mismatch for @effection/core here unless we provide this explicit resolution. I think this might be a deeper problem in how the semver constraints in effection are set up, but I'm not sure.